objects/src/transaction/compiled_tx.rs

```diff
@@ -0,0 +1,56 @@
+use super::{AccountId, Digest, Note, ProgramInfo, Vec};
+
+/// Resultant object of compiling a set of note scripts in the context of an account.
+/// The CompiledTransaction does not contain any account data, it is passed to a component
+/// that will do the required data lookups before processing the transaction further.
+/// Contains:
+/// - account_id: the account that the transaction was executed against.
+/// - consumed_notes: a list of consumed notes.
+/// - tx_script_root: optional MAST root for the transaction script.
+/// - tx_program: an executable program describing the transaction.
+pub struct CompiledTransaction {
+    account_id: AccountId,
+    consumed_notes: Vec<Note>,
+    tx_script_root: Option<Digest>,
+    tx_program: ProgramInfo,
```

Comment:
This should be not `ProgramInfo` but `Program` struct (from [here](https://github.com/0xPolygonMiden/miden-vm/blob/main/core/src/program/mod.rs#L26)).

Reply:
updated